Mjml.Net/Components/Body/TableComponent.cs

```diff
@@ -0,0 +1,118 @@
+namespace Mjml.Net.Components.Body
+{
+    public partial class TableComponent : Component
+    {
+        public override string ComponentName => "mj-table";
+        public override ContentType ContentType => ContentType.Raw;
```

Comment:
Please one line between the members  ... I am so picky, I know :D

Reply:
All good, I'll change this now :)

Reply:
@SebastianStehle, I actually really like the comments. It helps me improve so I'm always open to critic!